.github/workflows/unit-test-tools.yml

```
@@ -1,12 +1,12 @@
-name: unit-tests-diff-processor
+name: unit-tests-tools
```

tests -> test for consistency

```suggestion
name: unit-test-tools
```